src/Giveaway.js

```diff
@@ -449,6 +449,7 @@ class Giveaway extends EventEmitter {
                 this.channel.send(options.messages.congrat
                     .replace('{winners}', formattedWinners)
                     .replace('{messageURL}', this.messageURL)
+                    .replace('{prize}', this.prize);
```

Comment:
Remove semicolon ";" and move the line one line above so it matches the `end()` function